mmv1/products/privateca/Certificate.yaml

```diff
@@ -61,6 +61,10 @@ examples:
       ca_pool_id: 'my-pool'
     test_env_vars:
       project: :PROJECT_NAME
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'privateca_certificate_generated_key'
+    primary_resource_id: 'default'
+    skip_test: true
```

Comment:
please also add `skip_docs: true` to the privateca_certificate_config example - unless you think having both is valuable, but the existing examples are already pretty long: https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/privateca_certificate

it looks like the filebase64 usage is also part of the "with template example" so I think there wouldn't be much information loss?

If using TLS is the preferred path it might make sense to put it earlier in the list?

Reply:
Done.